packages/react-components/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -115,17 +127,16 @@ const useRootStyles = makeStyles({
   },
   'filled-warning': {
     backgroundColor: tokens.colorPaletteYellowBackground3,
-    color: tokens.colorNeutralForeground1,
-    ...shorthands.borderColor(tokens.colorPaletteYellowBackground3),
+    color: tokens.colorNeutralForeground1Static,
```

Comment:
The text isn't visible in high contrast ([link to Screener](https://screener.io/v2/states/4V5IW.microsoft-fluentui-react-components/badge-font-styles/1024x768/Chrome/badge-converged-tint-high-contrast-0)). 

I rejected the change but this is also an issue in the baseline snapshot so it doesn't appear this change introduced the issue. I'm okay with approving the change if you want to fix this in a follow up PR.

<img width="1335" alt="tint_high_contrast" src="https://user-images.githubusercontent.com/93940821/172736374-37c54bfb-4ceb-4eb5-b676-2766c9a797fa.png">

Reply:
Actually it might be a bug with the Badge spec, not the tokens, since it's using mismatched YellowBackground1 and YellowForeground2. Logged https://github.com/microsoft/fluentui/issues/23475 to follow up.

Reply:
Sounds good. I approved the Screener change.